app/Console/Commands/Setup/SetupSyncPteroCommand.php

```diff
@@ -59,6 +60,16 @@ public function handle()
         Egg::syncEggs();
         $bar->advance();
 
+        $this->newLine();
+        $this->line('Syncing Servers..this might take a while.');
+
+        Server::inRandomOrder()->chunk(15, function($servers) {
+            foreach ($servers as $server) {
+                Server::syncPterodactylServerSpecs($server);
```

Comment:
Why the * is this a static method which provides it self as parameter, the fuck?

Reply:
uuhm what?